examples/gluon_alexnet/README.md

```diff
@@ -0,0 +1,370 @@
+This is a short tutorial on how to write a custom Gluon model, and then serve it with MMS.
+
+This tutorial covers the following:
+1. [Prerequisites](#prerequisites)
+2. [Serve a Gluon model](#load-and-serve-a-gluon-model)
+  * [Load and serve a pre-trained Gluon model](#load-and-serve-a-pre-trained-gluon-model)
+  * [Load and serve a custom Gluon model](#load-and-serve-a-custom-gluon-imperative-model)
+  * [Load and serve a custom hybrid Gluon model](#load-and-serve-a-hybrid-gluon-model)
+3. [Conclusion](#conclusion)
+
+## Prerequisites
+
+* **Basic Gluon knowledge**. If you are using Gluon for the first
+time, but are familiar with creating a neural network with MXNet or another framework, you may refer this 10 min Gluon crash-course: [Predict with a pre-trained model](http://gluon-crash-course.mxnet.io/predict.html).
+* **Gluon naming**. Fine-tuning pre-trained Gluon models requires some understanding of how the naming conventions work. Take a look at the [Naming of Gluon Parameter and Blocks](https://mxnet.incubator.apache.org/tutorials/gluon/naming.html) tutorial for more information.
+* **Basic MMS knowledge**. If you are using MMS for the first time, you should take advantage of the [MMS QuickStart tutorial](https://github.com/awslabs/mxnet-model-server#quick-start).
+* **MMS Custom Service knowledge.** Review the [Defining a Custom Service](https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md) documentation to understand how MMS implements custom pre and post-processing for models.
+* **MMS installed**. If you haven't already, [install MMS with pip](https://github.com/awslabs/mxnet-model-server/blob/master/docs/install.md#install-mms-with-pip) or [install MMS from source](https://github.com/awslabs/mxnet-model-server/blob/master/docs/install.md#install-mms-from-source-code). Either installation will also install MXNet.
+
+
+Refer to the [MXNet model zoo](https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html) documentation for examples of accessing other models.
+## Load and serve a Gluon model
+
+There are three scenarios for serving a Gluon model with MMS:
+
+1. Load and serve a pre-trained Gluon model.
+2. Load and serve a custom imperative Gluon model.
+3. Load and serve a custom hybrid Gluon model.
+
+To learn more about the differences between gluon and hybrid gluon models refer to [the following document](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html)
+
+### Load and serve a pre-trained Gluon model
+
+Loading and serving a pre-trained Gluon model is the simplest of the three scenarios. These models don't require you to provide `symbols` and `params` files.
+
+While it is easy to access a model with a couple of lines of code, with MMS you will want to use a [MMS custom service](https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md) code pattern as follows:
+
+```python
+class MMSPretrainedAlexnet(MXNetVisionService):
```

Comment:
Can you please add comments to the code explaining what's going on? e.g. we define a service class that extends the existing vision service class, we're normalizing the image, etc.